ManageTeamsIdentityMobileAndDesktop/ManageTeamsIdentityMobileAndDesktop.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
consider changing this to `disable` or fixing the warnings generated by `Environment.GetEnvironmentVariable`